SimpleGame/src/main/scala-2.11/edu/gemini/experimental/jluhrs/Game.scala

```diff
@@ -0,0 +1,92 @@
+package edu.gemini.experimental.jluhrs
+
+import EventStream._
+import Timer._
+import scalaz._
+import Scalaz._
+
+/**
+  * Created by jluhrs on 5/24/16.
+  */
+
+/* The game logic is here.
+ * It's just an object that moves at constant speed, and refresh its position at given times.
+ * The direction of movement is affected by some input events.
+ */
+object Game {
+
+  type Position = (Double, Double)
+  type SpeedDir = (Double, Double)
```

Comment:
I know this is just a toy exercise but `case class`es to give these two concepts distinct types and make clear what they contain would be useful.